torch/testing/_internal/common_utils.py

```diff
@@ -1726,6 +1726,51 @@ def wrapper(*args, **kwargs):
     return decorator
 
 
+def make_dynamo_test(
+    fn: Optional[Callable[..., Any]] = None, expected_frame_count: int = 1
+) -> Callable[..., Any]:
+    """
+    Decorator function to create a dynamo test case.
+
+    Dynamo injects a dummy call to `torch.sin(random_tensor)` to ensure graph
+    generation, even when testing for side effects.
```

Comment:
Since we're here now and https://github.com/pytorch/pytorch/pull/146527 has landed, there's no need for the .sin calls anymore